src/lib/Libnet/net_server.c

```diff
@@ -497,10 +532,9 @@ engage_authentication(conn_t *pconn)
  *
  */
 int
-wait_request(time_t waittime)
+wait_request(time_t waittime, struct sched_socks scks)
```

Comment:
As said earlier please use generic name instead of struct sched_socks.

Reply:
Also you can just pass a pointer to the structure. This we can just pass NULL while second parameter is not applicable.

Reply:
done